roles/rhods_test_jupyterlab/tasks/main.yml

```diff
@@ -176,6 +176,27 @@
     oc delete ev -n {{ rhods_test_namespace }} --all > /dev/null
   failed_when: false
 
+- name: Update default notebook size
+  environment:
+    KUBECONFIG: '{{ sut_cluster_kubeconfig }}'
+    NOTEBOOK_RAM: '{{ rhods_test_jupyterlab_notebook_memory }}'
+    NOTEBOOK_CPU: '{{ rhods_test_jupyterlab_notebook_cpu }}'
+
+  command: |
+    oc get configmap -o yaml --namespace redhat-ods-applications odh-jupyterhub-global-profile > /tmp/odh-jh-gp.yaml
+    # edit the default notebook config map for RHODS
+    cat /tmp/odh-jh-gp.yaml | \
+      yq -y .memory=\"${NOTEBOOK_RAM}Gi\" | \
+      yq -y .cpu=\"${NOTEBOOK_CPU}\" | \
+        > /tmp/odh-jh-gp-ours.yaml
+    # also create a new config map with the previous values.
+    # TODO: upload it.
+    cat /tmp/odh-jh-gp.yaml | \
+      yq -y .name=\"odh-jupyterhub-global-profile-orig\" \
+        > /tmp/odh-jh-gp-orig.yaml
+    oc create configmap --namespace redhat-ods-applications /tmp/odh-jh-gp-ours.yaml
```

Comment:
```
oc create configmap --namespace redhat-ods-applications /tmp/odh-jh-gp-ours.yaml
```
* typo: `-f` required
* `--namespace ...` not necessary, because the json already contains it

can be simplified into (and needs to be json):
```
   cat /tmp/odh-jh-gp.yaml | \
      yq -y .name=\"odh-jupyterhub-global-profile-orig\" |\
      oc create configmap -f-
```

Reply:
Done